01-Authorization/auth0authorization/views.py

```diff
@@ -25,33 +13,29 @@ def requires_scope(required_scope):
     """
     def require_scope(f):
         @wraps(f)
-        def decorated(*args, **kwargs):
-            token = get_token_auth_header(args[0])
-            decoded = jwt.decode(token, verify=False)
+        def decorated(request, *args, **kwargs):
+            decoded = jwt.decode(request.auth, options={'verify_signature': False})
```

Comment:
`JSONWebTokenAuthentication` returns the token as part of the authentication tuple: `request.auth` holds the token after successful authentication.